src/strings/en-US.json

```diff
@@ -138,7 +138,7 @@
   "ButtonShuffle": "Shuffle",
   "ButtonShutdown": "Shutdown",
   "ButtonSignIn": "Sign In",
-  "ButtonSignOut": "Sign out",
+  "ButtonSignOut": "Sign Out",
   "ButtonSignUp": "Sign up",
```

Comment:
```suggestion
  "ButtonSignUp": "Sign Up",
```

Reply:
I think this needs to be removed anyway, it was probably only used for connect.